src/build/checkstyle.xml

```diff
@@ -1,105 +1,136 @@
+        <module name="SimplifyBooleanExpression"/>
+        <module name="SimplifyBooleanReturn"/>
+        <module name="HideUtilityClassConstructor"/>
+        <module name="InterfaceIsType"/>
+        <!--<module name="VisibilityModifier"/>-->
+        <module name="ArrayTypeStyle"/>
+        <module name="TodoComment"/>
+        <module name="UpperEll"/>
+        <module name="NeedBraces"/>
+        <module name="AvoidStarImport"/>
+        <module name="ConstantName"/>
+        <!--<module name="DeclarationOrder"/>-->
+        <module name="EqualsAvoidNull"/>
+        <module name="HiddenField">
+            <property name="ignoreConstructorParameter" value="true"/>
+            <property name="ignoreSetter" value="true"/>
+            <property name="setterCanReturnItsClass" value="true"/>
+            <property name="ignoreAbstractMethods" value="true"/>
         </module>
-		<module name="ModifierOrder"/>
-		<!--<module name="RedundantModifier"/>-->
-		<!--<module name="EmptyBlock"/>-->
-		<module name="LeftCurly"/>
-		<module name="NeedBraces"/>
-		<module name="RightCurly"/>
-		<module name="EmptyStatement"/>
-		<module name="EqualsHashCode"/>
-		<module name="IllegalInstantiation"/>
-		<!--<module name="InnerAssignment"/>-->
-		<module name="MissingSwitchDefault"/>
-		<module name="SimplifyBooleanExpression"/>
-		<module name="SimplifyBooleanReturn"/>
-		<module name="HideUtilityClassConstructor"/>
-		<module name="InterfaceIsType"/>
-		<!--<module name="VisibilityModifier"/>-->
-		<module name="ArrayTypeStyle"/>
-		<module name="TodoComment"/>
-		<module name="UpperEll"/>
-		<module name="NeedBraces"/>
-		<module name="AvoidStarImport"/>
-		<module name="ConstantName"/>
-		<!--<module name="DeclarationOrder"/>-->
-		<module name="EqualsAvoidNull"/>
-		<!--<module name="HiddenField"/>-->
-		<module name="InnerTypeLast"/>
-		<module name="MissingDeprecated"/>
-		<module name="MissingOverride"/>
-		<module name="MutableException"/>
-		<module name="NoClone"/>
-		<module name="NoFinalizer"/>
-		<module name="OneStatementPerLine"/>
-		<module name="OneTopLevelClass"/>
-		<module name="OuterTypeFilename"/>
-		<module name="ParameterAssignment"/>
-		<module name="StringLiteralEquality"/>
-		<module name="SuperClone"/>
-		<module name="SuperFinalize"/>
-		<!--<module name="TrailingComment"/>-->
-		<module name="VariableDeclarationUsageDistance"/>
+        <module name="InnerTypeLast"/>
+        <module name="MissingDeprecated"/>
+        <module name="MissingOverride"/>
+        <module name="MutableException"/>
+        <module name="NoClone"/>
+        <module name="NoFinalizer"/>
+        <module name="OneStatementPerLine"/>
+        <module name="OneTopLevelClass"/>
+        <module name="OuterTypeFilename"/>
+        <module name="ParameterAssignment"/>
+        <module name="StringLiteralEquality"/>
+        <module name="SuperClone"/>
+        <module name="SuperFinalize"/>
+        <module name="TrailingComment"/>
```

Comment:
I'm on the fence with this one, as it disrupts the call-out markers in the documentation code.

Reply:
This was also one of my least favorites